packages/core/src/react-integration/__tests__/integration.web.tsx

```diff
@@ -39,7 +39,9 @@ afterEach(() => {
     removeEventListener('error', onError);
 });
 
-for (const makeProvider of [makeCacheProvider, makeExternalCacheProvider]) {
+for (const makeProvider of [
+  makeCacheProvider /*, makeExternalCacheProvider*/,
```

Comment:
why is this commented out?

Reply:
ack! was putting console logs in the test and didn't want to get spammed. great catch!